### DISCUSSION TITLE
Onboarding "view branch" timing

### DISCUSSION THREAD

**Discussion by croaky** [planetscale/discussion#13] [Bug Reports] [answered]
For a short period of time (10 seconds-ish?), before the onboarding branch is ready, if I click "View branch", I get a 404. Maybe that button could be disabled with a message like "Creating branch..." until the branch is ready? 

https://user-images.githubusercontent.com/198/118693889-56a21a00-b7c0-11eb-8434-38ebacf935cb.mov

**Comment by aquarapid** [comment]
Agreed!  The branch launch latency can be a bit variable depending on the state of the backend infra;  we hope to get it down to somewhat below 10 seconds;  but blocking the UI element temporarily is a great idea!

**Comment by hallaroo** [comment, marked as answer]
Just an update here. This should now be fixed and the button has now been disabled (just as you suggested). Thank you so very much for the report!

**Reply by croaky** [reply]
🎉 Awesome!
